docs/JestObjectAPI.md

```diff
@@ -555,10 +555,6 @@ test('works too', () => {
 
 Returns the `jest` object for chaining.
 
-### `jest.isEnvironmentTornDown(fn)`
```

Comment:
Moved to [Misc](https://jestjs.io/docs/next/jest-object#misc) section instead of Mock Modules.